scripts/superdesk-archive/views/upload.html

```diff
@@ -23,7 +23,7 @@ <h3 class="pull-left">{{ activity.label|translate }}</h3>
         </div>
         <ul class="upload-thumbs">
             <li ng-repeat="item in items">
-                <div class="thumb" sd-image-preview="item.url" data-file="item.file">
+                <div class="thumb" sd-image-preview="item.url" data-file="item.file" data-width="item.width" data-height="item.height">
```

Comment:
(As we mentioned in Slack) Img srcs here are created from local file Blobs turned into a dataUrl. Might as well set the height/width of the image when we've created this dataURL here.